src/elements/icon/icon.js

```diff
@@ -0,0 +1,42 @@
+(function() {
+  'use strict';
+
+  angular
+  .module('semantic.ui.elements.icon', [])
+  .directive('smIcon', smIcon);
+
+
+  function smIcon() {
+    return {
+      restrict: 'E',
+      replace: true,
+      transclude: true,
+      scope: {
+        fontIcon: '@smFontIcon',
+        iconSize: '@smIconSize'
```

Comment:
Please, remove `@smIconSize`parameter. Semantic-UI has its own [set of classes](http://semantic-ui.com/elements/icon.html#size) for sizes which we will let them use as needed.